main.py

```diff
@@ -0,0 +1,124 @@
+    "B": adv_b,
+    "C": adv_c
+}
+
+#This is the primary function that you start at when you begin the game
+def intro():
+    adventures = 1 #we start with only 1 adventure availible
+    say("""
+    Welcome brave adventurer! You have 3 different adventures to depart on. 
+    The adventures must be completed in order! To begin we need to know about you!
+    """)
+    # we create a character with a name and pronouns
+    protag = character_creation()
+    say("Let us begin!")
+    # let our user pick their adventure
+    success = pick_adventure(get_options(adventures))
+    adventures += success #if the protag is successful they get another adventure
+    while True:
+        say("Would you like to go on an adventure?")
+        choice = input("(y/n)")
+        if choice == "y" or "Y":
+            success = pick_adventure(get_options(adventures))
+            if adventures < ADVENTURE_FUNCTIONS.count():
+                adventures += success
+        else:
+            break
+    say("It has been great adventuring with you, " + protag.name)
+
+#this function takes in a number of adventures avaible to the user
+#returns a dictionary of different adventures they can go on
+def get_options(adventures):
+    available_adventures = {}
+    all_adventures = list(ADVENTURES.keys())[0:adventures]
+    for adventure in all_adventures:
+        available_adventures[adventure] = ADVENTURES[adventure]
+    available_adventures["E"] = "Exit"
+    return available_adventures
+
+# this function takes in the adventures our user can go on 
+# this function returns 1 on a sucessful adventure and 0  on a failed adventure or exit
+def pick_adventure(options):
+    say("Pick an adventure below by entering the letter associated")
+    # To Do Replace Adventure X with the correct name of the adventure
+    print(options)
+    while True:
+        for key,adventure_name in options.items():
+            say(key + ':' + adventure_name)
+        choice = input()
+        if choice in ADVENTURE_FUNCTIONS.keys():
+            return ADVENTURE_FUNCTIONS[choice]()
+        elif choice == "E":
+            return 0
+        else:
+            say("That is not an option. Please enter only the letter for the adventure")
+
+# this function take in no variables
+# this returns a Protagonist object with a name and pronouns 
+def character_creation():
+    name = check_name()
+    say("You say your name is " + name + " ?\n This is the name of the adventurer of legend!")
+    pronouns = check_pronouns()
+    protag = Protagonist(name, pronouns)
+    return protag
+
+# this function takes in no inputs
+# this function returns a list of pronouns e.g.['he', 'him', 'his']
+def check_pronouns():
+    while True:
+        say("""
+        Brave adventurer, what are your pronouns?
+        """)
+        say("(Please format as follows: he/him/his)")
+        pronouns = input("Pronouns: ")
+        subject_pronoun, object_pronoun, possessive_pronoun = pronouns.split('/')
```

Comment:
I would put some sort of error handling here in case they don't format the pronouns correctly.
Something like this:
```
try:
    subject_pronoun, object_pronoun, possessive_pronoun = pronouns.split('/')
    say("Does this sound correct? \'" + subject_pronoun +" has " + possessive_pronoun + " sword with " + object_pronoun + "\'")
    correct = input("(y/n) ")
    if correct == "y":
        return [subject_pronoun, object_pronoun, possessive_pronoun]
    else:
        say("My mistake, let us try again\n")
except ValueError: # Error handling in case user doesn't format pronouns correctly
    say("Sorry, that formatting doesn't look right.")
```

Reply:
Thanks for catching that! I'll fix it right away